pkg/declarativeconfig/uuid.go

```diff
@@ -0,0 +1,34 @@
+package declarativeconfig
+
+import "github.com/stackrox/rox/pkg/uuid"
+
+const (
+	authProviderNS  string = "auth-provider"
```

Comment:
nit: should we call them namespaces? Then they can be confused with k8s namespaces. Perhaps prefix is a better word?

Reply:
Hm, I'm a bit on the fence with this one. While it may be true, in the context of UUID generation, namespace _is_ the correct name. Prefix would be something we currently do with our internal IDs[ such as the groups ID generation](https://github.com/stackrox/stackrox/blob/65820256abbcb6cc2809f7a6ece8ca7729dd904a/central/group/datastore/validate.go#L53-L56), so I'd rather stick with namespace for now as it's more explicit in the current context of the file.

Reply:
One option to reduce the confusion would be to name these namespaces UUID Namespaces.
An alternative I could see would be to put a comment stating that the namespaces used here are UUID namespaces and not k8s namespaces.

Reply:
UUID Namespace sounds good. Anyway, I'll leave it up to @dhaus67 to decide